experiments/orbital_discriminability/gnss_independent_pair_qualification.py

```
@@ -0,0 +1,1078 @@
+            ),
+            "reason": str(exc),
+            "clause_states": {
+                "artifact_materialization_and_hash": (
+                    "SATISFIED" if len(artifacts) == len(PRODUCTS) else "UNSATISFIED"
+                ),
+                "measurement_qualification": "UNSATISFIED",
+                "primary_orbital_comparison": "NOT_EVALUATED",
+            },
+            "next_authority": "NONE_ALGO_MDO_ROLE_CLOSED",
+        }
+    except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)